### PR TITLE
Add participants count

### PR DESCRIPTION
**Pull request by vp8x8**
Add a new (clickable) button showing the participants count number next to the subject.
Clicking on it should open the Speaker stats dialog.

### CODE REVIEW

**Comment by damencho** [comment]
What do JIT-3394 means? Can you give some meaningful description for this PR? And please do not add private links to the public accessible resources, they have no value here. Thank you.

**Comment by vp8x8** [comment]
@damencho Sorry, I'm new to this. It was the link to a jira ticket. I'll keep that in mind. Updated the description.

**Comment by damencho** [comment]
Thank you! :) We are discussing offline future changes for jira integration and we are still experimenting with it. And when we have something official we will let you know. Thank you for the quick reaction.

**Comment by vp8x8** [comment]
@saghul Asked for a Screen shot. Here it is:

shot

**Comment by yanas** [comment]
@vp8x8 can you please also add a screen shot with a conference subject? Thank you!

File: react/features/conference/components/web/Subject.js [hunk 1]
```diff
@@ -2,9 +2,12 @@
 
 import React, { Component } from 'react';
 
+import ParticipantsCount from './ParticipantsCount';
```
Comment (hristoterezov) [inline react/features/conference/components/web/Subject.js]: The group of imports from the same feature should be placed after the group with imports from other features.

File: react/features/conference/components/web/ParticipantsCount.js [hunk 2]
```diff
@@ -0,0 +1,101 @@
+// @flow
+
+import React, { PureComponent } from 'react';
+import type { Dispatch } from 'redux';
+
+import { connect } from '../../../base/redux';
+import { getParticipantCount } from '../../../base/participants';
+import { openDialog } from '../../../base/dialog';
```
Comment (hristoterezov) [inline react/features/conference/components/web/ParticipantsCount.js]: Order the imports alphabetically by filename.

**Comment by vp8x8** [comment]
@yanas 
subject

**Comment by hristoterezov** [comment]
This part LGTM! Although I agree with @saghul . 

@yanas should I merge the PR as it is or you'd like @saghul 's comments to be addressed first?

**Comment by yanas** [comment]
> Generally LGTM, module Hristo's suggestions. General question (for @yanas I guess): do we want to show that number on mobile too? We don't have speaker stats there though.
> 
I do think it would be nice to have it on mobile too.

> On a subjective note, I don't like how the design for this feature looks like (not your fault @vp8x8): when no sobject is set it looks "dangling" or floaing on top of the screen. I find that it clutters the screen and brings us back to when there were 2 toolbars, one in that exact position.

I have expressed the same concern, but Emil convinced me it's good enough first version and we can start from there.

**Comment by hristoterezov** [comment]
@saghul We've agreed with @yanas to merge this PR and address the mobile part trough another PR.

**Pull request merged**